src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -1140,13 +1140,15 @@ private void ThrowIfWebhooksNotConfigured()
 
         internal bool TryGetRpcBaseUrl(out Uri rpcBaseUrl)
         {
-            if (this.durableTaskOptions.LocalRpcEndpointEnabled != false)
+            if (this.durableTaskOptions.LocalRpcEndpointEnabled != false
+                && this.localHttpListener.IsListening)

```

Comment:
nit: I would prefer if we didn't introduce the double negative with "!= false"
```suggestion
            if (this.durableTaskOptions.LocalRpcEndpointEnabled
                && this.localHttpListener.IsListening)

```

Reply:
Unfortunately the two expressions are not equivalent because `LocalRpcEndpointEnabled` is not a bool, but a bool? (i.e. `Nullable<Boolean>`). In our case, `true`, `false`, and `null` all have different semantic meanings.

Reply:
Ah, that makes sense. Thanks for clarifying!